packages/backend-modules/voting/lib/Submission.js

```diff
@@ -80,20 +78,41 @@ const createSubmissionsQuery = ({
     terms: { userId: filters.userIds },
   }
   const mustQuestionnaireId = questionnaireId && { term: { questionnaireId } }
+  const mustSubmissionId = filters?.id && { term: { id: filters.id } }
+  const mustNotSubmissionId = filters?.not && { term: { id: filters.not } }
```

Comment:
Both `mustNotSubmissionId` and `mustNotSubmiddionIds` are in `must_not` (line 194), but here the filter is already defined as `filters?.not`, wouldn't that be a double negation?

Reply:
Oh, I see were you went there. `filters.not` is just a prop name, like `filters.foobar`. It has no impact on negation.